core/src/main/resources/hudson/node_monitors/DiskSpaceMonitorNodeProperty/config.jelly

```diff
@@ -0,0 +1,15 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <f:entry field="freeDiskSpaceThreshold" title="${%Free Disk Space Threshold}">
+    <f:textbox default="1GiB"/>
```

Comment:
When I tried to enter a value with "GiB" suffix, a stack trace was displayed showing:
```
java.lang.NumberFormatException: For input string: "60GI"
	at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
	at java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
	at java.base/java.lang.Double.parseDouble(Double.java:792)
	at hudson.node_monitors.DiskSpaceMonitorDescriptor$DiskSpace.parse(DiskSpaceMonitorDescriptor.java:163)
	at hudson.node_monitors.AbstractDiskSpaceMonitor.<init>(AbstractDiskSpaceMonitor.java:23)
	at hudson.node_monitors.TemporarySpaceMonitor.<init>(TemporarySpaceMonitor.java:54)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.kohsuke.stapler.RequestImpl.invokeConstructor(RequestImpl.java:604)
	at org.kohsuke.stapler.RequestImpl.instantiate(RequestImpl.java:883)
	at org.kohsuke.stapler.RequestImpl$TypePair.convertJSON(RequestImpl.java:768)
Caused: java.lang.IllegalArgumentException: Failed to instantiate class hudson.node_monitors.TemporarySpaceMonitor from {"freeSpaceThreshold":"60GiB","freeSpaceWarningThreshold":"2GB"}
```

I think it is better to show the misleading suffix.

Reply:
> from the stacktrace it looks as if you tested this not with my change. The line numbers indicate that this is the old version.

That must have been what happened.  I'll try it again later today.

Reply:
Confirmed that you are correct @mawinter69 .  I was using a half-baked configuration because I had failed to rebuild after switching context from one pull request to the next.  Resolving